projects/vdk-heartbeat/src/vdk/internal/heartbeat/job_controller.py

```diff
@@ -233,7 +244,7 @@ def set_job_property(self, key: str, value: str):
         )
 
     @LogDecorator(log)
-    def enable_deployment(self):
+    def enable_deployment_and_update_vdk_version(self):
```

Comment:
here is says "enable_deployment_and_update_vdk_version" - but this only enables it. 

While `disable_deployment` has the ` + __get_vdk_version_arg`

am I missing something?

Reply:
I don't know how that happened, I definitely tested it in enable_deployment_and_update_vdk_version, since running it now in disable_deployment has a totally different behavior. Maybe I did some mix-up when committing. Anyway, I changed it. Thank you!